scripts/akrtma3d/exakrtma3d_gsianl.ksh

```diff
@@ -192,7 +192,7 @@ nummem=`more filelist03 | wc -l`
 nummem=$((nummem - 3 ))
 hrrrmem=`more filelist.hrrrdas | wc -l`
 hrrrmem=$((hrrrmem - 3 ))
-if [[ ${hrrrmem} -gt 30 ]] && [[ ${HRRRDAS_BEC} -eq 1  ]]; then #if HRRRDAS BEC is available, use it as first choice
+if [[ ${hrrrmem} -gt 30 ]] && [[ ${HRRRDAS_BEC} -ge 1  ]]; then #if HRRRDAS BEC is available, use it as first choice
```

Comment:
This is just a general comment, not related to this PR. Why is this seciton included here if there's no HRRRDAS for Alaska?

Reply:
Hi @MatthewMorris-NOAA 
You are right. This part is actually not necessary for current Alaska run. We can remove it in the later PR.
-Gang